templates/pubspecs/packages/relic/pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: relic
+description: A lightweight web server inspired by Shelf.
+version: SERVERPOD_VERSION
```

Comment:
Suggestion: Remove the serverpod version from the relic package. This should be managed separately.

Reply:
Or even better, add a RELIC_VERSION variable here that is substituted from a separate file in the root.

Reply:
Ah, a didn't see this when doing the review. A `RELIC_VERSION` would be 👌. Perhaps this is easier left to a follow up PR though?

Reply:
Yes, maybe just remove all templating for now.